node/storage-cleaner/README.md

```diff
@@ -0,0 +1,54 @@
+# Node.js Storage Cleaner Function
+
+Storage cleaner function to remove all files from the buckets older than X number of days.
+
+## 🧰 Usage
+
+### GET /
+
+Remove the files older than X number of days from all buckets.
+
+**Response**
+
+Sample `200` Response: Buckets cleaned
+
+## ⚙️ Configuration
+
+| Setting           | Value         |
+| ----------------- | ------------- |
+| Runtime           | Node (18.0)   |
+| Entrypoint        | `src/main.js` |
+| Build Commands    | `npm install` |
+| Permissions       | `any`         |
+| CRON              | `0 1 * * *`   |
+| Timeout (Seconds) | 900           |
+
+## 🔒 Environment Variables
```

Comment:
Let's introduce `APPWRITE_BUCKET_ID` env variable, and let's only delete files from that one specific bucket.

Reply:
done